tests/e2e/Services/Teams/TeamsBaseServer.php

```diff
@@ -59,8 +61,8 @@ public function testCreateTeamMembership($data): array
         $this->assertNotEmpty($response['body']['userId']);
         $this->assertNotEmpty($response['body']['teamId']);
         $this->assertCount(2, $response['body']['roles']);
-        $this->assertIsInt($response['body']['joined']);
-        $this->assertGreaterThan(0, $response['body']['joined']);
+        $this->assertEquals(true, DateTime::isValid($response['body']['joined']));
+        $this->assertGreaterThan('2020-07-07 07:58:44', DateTime::now());
```

Comment:
I think we don't need this assert anymore after we validated its a DateTime 👍🏻

Reply:
ok